.editorconfig

```diff
@@ -0,0 +1,14 @@
+# All files
+[*]
+end_of_line              = crlf
+indent_style             = space
+insert_final_newline     = true
+trim_trailing_whitespace = true
+# Project files
+[*.{csproj,fsproj}]
+indent_size = 2
+charset     = utf-8-bom
+# Code files
+[*.{cs,csx,fs,fsx,vb,vbx,xml}]
+indent_size = 4
+charset     = utf-8-bom
```

Comment:
Why BOM instead of just utf-8?

Reply:
Usually, I've seen this option set to BOM